bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -37,6 +37,13 @@
 public abstract class AbstractConfiguration extends CompositeConfiguration {
 
     static final Logger LOG = LoggerFactory.getLogger(AbstractConfiguration.class);
+    public static final String READ_SYSTEM_PROPERTIES_PROPERTY
+                            = "org.apache.bookkeeper.conf.readsystemproperties";
```

Comment:
Another candidate would be "org.apache.bookkeeper.conf.usesystemproperties". This system property is used both for servers (Bookie) and clients